Makefile

```diff
@@ -383,6 +379,10 @@ uninstall:
 	  echo rm -f $(DESTDIR)$(bindir)/`basename $$f`; \
 	  rm -f $(DESTDIR)$(bindir)/`basename $$f`; \
 	done
+	@for f in $(PKGLIBEXEC_PROGRAMS); do \
+	  echo rm -f $(DESTDIR)$(pkglibexecdir)/`basename $$f`; \
```

Comment:
Not too familiar with install targets and makefiles in general, but why echo the command and then execute it in the next line? Wouldn't `make uninstall` echo executed commands anyway?

Reply:
We are iterating over the files using `for`. It is easier to understand individual `rm` commands than a complex `for` statement. So we instead suppress printing the `for` statement and print the individual `rm` commands executed inside the `for` loop.

Reply:
And the `for` is being suppressed by the leading `@`, great. Thanks for the explanation.